### PR TITLE
fix(ci): remove website deployment from production branch ci

### PR DESCRIPTION
**Pull request by clemsos**
# Description

The settings were incorrect, also we dont deploy from `production` branch anymore

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -46,9 +46,9 @@ jobs:
       target-env: staging
```
Comment (julien51) [inline .github/workflows/master.yml:46]: i think this should say `prod` ?

Comment (searchableguy) [reply .github/workflows/master.yml:46]: Yes

**Comment by clemsos** [comment]
ok that one can be merged as well

**Review by julien51 — APPROVED** [review]

**Comment by julien51** [comment]
🚢

**Pull request merged**